notebooks/ImvizExample.ipynb

```diff
@@ -624,10 +624,51 @@
     "imviz.link_data(link_type='wcs')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7a4f7c3f",
+   "metadata": {},
+   "source": [
+    "If you want a second viewer, click on the \"picture with +\" icon on top. You can also programmatically control the new viewer, as follows."
```

Comment:
We probably want code to do this programatically, but definitely not critical for this PR unless trivial.

Reply:
I couldn't figure out how to do this... it currently relies on callback that is generated interactively by the VueJS layer. Let's defer.

Reply:
Let's continue this one over at #882 .